index.bs

```diff
@@ -4900,6 +5008,83 @@ as candidates to be employed in a [=registration ceremony=].
 : Authenticator extension output
 :: None.
 
+
+## Credential Properties Extension (<dfn>credProps</dfn>) ## {#sctn-authenticator-credential-properties-extension}
+
+This [=client extension|client=] and [=authenticator extension|authenticator=] [=registration extension=] facilitates reporting certain [=credential properties=] to the requesting [=[WRP]=] upon creation of a [=public key credential source=] as a result of a [=registration ceremony=].
+
+Generally, this extension allows for both, or either, the [=authenticator=] or the [=client=] to report on [=credential properties=]. However, certain [=credential properties=], known as <dfn>source-specific credential properties</dfn>, are defined to be reported on by only the [=authenticator=] or by the [=client=].
+
+In the opposite case, known as <dfn>all-sources credential properties</dfn>, it is possible for either the [=authenticator=] or the [=client=] to report on a certain [=credential property=]. Additionally, the [=client=] MAY so report if the [=client=] determines the [=authenticator=] failed to report, and the [=client=] possesses the appropriate data to report.
+
+At this time, one [=credential property=], which is an [=all-sources credential property=], is defined. It is known abstractly as the <dfn>resident key credential property</dfn>. It is a boolean value indicating whether or not the {{PublicKeyCredential}} returned as a result of a [=registration ceremony=] is a [=resident credential=] or a [=non-resident credential=].  
+
+: Extension identifier
+:: `credProps`
```

Comment:
This also needs to be indicated in authenticatorGetInfo. May be open a seperate PR in FIDO2 specs or mention here that this identifier will be present in authenticatorGetInfo.

Reply:
good catch, thx. see https://github.com/fido-alliance/fido-2-specs/issues/660

Reply:
Removing authenticator extension component of the credProps extension per WG call on May 1, 2019 so this is now moot.